### PR TITLE
Fix block reward CI failure

### PR DESCRIPTION
**Pull request by ws4charlie**

### CODE REVIEW

**Comment by ws4charlie** [comment]
@kingpinXD Do you have time to take a look?

File: Makefile [hunk 1]
```diff
@@ -111,6 +111,7 @@ transfertest: reset
 rewardtest: reset
 	@./scripts/testsend
 	python scripts/reward/testRewards.py
+	make reset
```
Comment (kingpinXD) [inline Makefile:114]: do we need this becuase testRewards also creates wallet addresses ?

Comment (ws4charlie) [reply Makefile:114]: > do we need this becuase testRewards also creates wallet addresses ?

Yes. Both create wallet addresses. Without 'reset' the 2nd test somehow doesn't work well on the \keystore\ left by 1st test.

Comment (kingpinXD) [reply Makefile:114]: Yeah i noticed that as well, if we use the same pub /private key pair to generate two secure walllet accounts they stop working, 
Maybe we should look at it in a different ticket

Comment (ws4charlie) [reply Makefile:114]: Yep. Can you approve this one and let Mani merge CI first

**Pull request merged**